pennclubs/settings/base.py

```diff
@@ -146,6 +146,7 @@
 EMAIL_SUBJECT_PREFIX = '[Penn Clubs] '
 INVITE_URL = 'https://{domain}/invite/{club}/{id}/{token}'
 VIEW_URL = 'https://{domain}/club/{club}'
+EDIT_URL = 'https://{domain}/club/{club}/edit'
```

Comment:
Should this just use `reverse` from Django instead of setting it here?

Reply:
Reverse won't work because these pages are from the frontend, not the backend.

The domain is necessary because management scripts don't have a `request` object so they can't automatically detect the domain.